recipes-wpe/wpe/wpe_0.1.bb

```diff
@@ -186,4 +186,7 @@ RDEPS_MEDIASOURCE += "${RDEPS_EXTRA}"
 RDEPS_VIDEO += "${RDEPS_EXTRA}"
 RDEPS_WEBAUDIO += "${RDEPS_EXTRA}"
 
-RRECOMMENDS_${PN} += "ca-certificates"
+RRECOMMENDS_${PN} += " \
+    ca-certificates \
+    ttf-bitstream-vera \
```

Comment:
it would be better left to be added in image. Or may be create a virtual package for fonts. Since comcast images will use xfinity fonts. Its will force us to unnecessarily add a revert either via a bbappend or via blacklisting.